packages/plugin-core/src/components/lookup/utils.ts

```diff
@@ -707,6 +712,34 @@ export class NotePickerUtils {
     });
   };
 
+  /**
+   * Get picker results without input from the user
```

Comment:
nit: can we update this description? It's a little confusing - b/c there is actually "input" from the user, but it's just that b/c of limitations with how the quick pick control works, the callback didn't provide adequate info (If I'm understanding the issue correctly)

Reply:
How about

```suggestion
   * Get picker results with current picker value
```
and also rename the util to `fetchPickerResultsWithCurrentValue`?